execution_workflows/execution_output_specification.md

```diff
@@ -10,6 +10,8 @@ Outputs depend on features available for the participant (i.e. not all participa
   - BED file with identified poly(A) sites with single nucleotide resolution
 - quantification benchmarking event:
   - BED file with identified unique poly(A) sites and TPM values for each site
+  - BED file with identified unique poly(A) sites and relative usage values for each site
+  - BED file with coordinates of regions (e.g. terminal exons, genes) and the relative usage values for each region
```

Comment:
I'm afraid I don't quite understand what the "relative usage value" is for a region. Relative usage of distal site? Relative usage of the region compared to other regions? Can you elaborate a bit, also below in section "Format 05"?